src/main/scala/org/reactivecouchbase/rs/scaladsl/Bucket.scala

```diff
@@ -56,6 +57,8 @@ class Bucket(config: BucketConfig, onStop: () => Unit) {
     (_bucket, _bucket.async(), _bucketManager, Future.successful(_bucket.async()))
   }
 
+  private val durationToExpiry = (expiration:Duration) => ((if (expiration < 30.days) 0L else System.currentTimeMillis / 1000L) + expiration.toSeconds).toInt
```

Comment:
I think `Duration.Zero` is okay as it's how it works in Couchbase or even Memcached.

It's Okay for me, it's already merged and used on the last snapshot ;-)